File: python3/src/useCase/SellItemRequest.py
class SellItemRequest(object):
    def set_quantity(self, quantity: int):
        self.quantity = quantity

    def get_quantity(self):
        return self.quantity

    def set_product_name(self, product_name):
        self.product_name = product_name

    def get_product_name(self):
        return self.product_name

File: php/src/UseCase/SellItemRequest.php
<?php
declare(strict_types=1);

namespace Pitchart\TellDontAskKata\UseCase;

class SellItemRequest
{

    private string $productName;

    private int $quantity;

    public function __construct()
    {
    }

    /**
     * @param int $quantity
     * @return SellItemRequest
     */
    public function setQuantity(int $quantity): SellItemRequest
    {
        $this->quantity = $quantity;
        return $this;
    }

    /**
     * @return string
     */
    public function getProductName(): string
    {
        return $this->productName;
    }

    /**
     * @return int
     */
    public function getQuantity(): int
    {
        return $this->quantity;
    }

    public function setProductName(string $productName): self
    {
        $this->productName = $productName;
        return $this;
    }
}